### PR TITLE
So you wanna change your password kid?

### PR DESCRIPTION
**Pull request by LarrySchultheis**

### CODE REVIEW

File: rockr/views.py [hunk 1]
```diff
@@ -79,6 +79,12 @@ def delete_user_accout():
     data = uq.delete_user_account(request.args.get("id"), request.args.get("email"))
     return format_response(200, data)
 
+@app.route('/change_password', methods=["POST"])
+def change_password():
+    resp = uq.change_password(request.json)
+    print(resp)
```
Comment (squaill) [inline rockr/views.py]: 🤔

**Review by squaill — APPROVED** [review]
remove print pls

File: rockr/queries/user_queries.py [hunk 2]
```diff
@@ -48,6 +48,11 @@ def delete_user_account(user_id, email):
     api_wrapper.delete_auth0_account(email)
     return "success"
 
+def change_password(user):
+    print(user)
```
Comment (squaill) [inline rockr/queries/user_queries.py]: 👁👄👁

**Comment by LarrySchultheis** [comment]
Listen

**Pull request merged**